rust/scripts/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Run Go tests and CT for evmrs on branch origin/evmrs-main.
+# This script must be called from the parent directory of Tosca.
+
+mkdir -p ~/tests
```

Comment:
please dont create folders in users home.

Reply:
will just replace `~` with `..`